packages/web-api/src/WebClient.ts

```diff
@@ -144,10 +144,20 @@ export class WebClient extends EventEmitter<WebClientEvent> {
       throw new TypeError(`Expected an options argument but instead received a ${typeof options}`);
     }
 
-    const response = await this.makeRequest(method, Object.assign(
-      { token: this.token },
-      options,
-    ));
+    // Sending an encrypted value in Authorization header instead of sending them in request body
+    let headers = {};
+    if (typeof options !== 'undefined'
+      && typeof options.client_id !== 'undefined'
+      && typeof options.client_secret !== 'undefined') {
+      const credentials = Buffer.from(`${options.client_id}:${options.client_secret}`).toString('base64');
```

Comment:
I believe this functionality belongs in the `serializeApiCallOptions()` method. all of the serialization-related logic is already there. that method is used with `axios`'s `transformRequest` pipeline and allows making modifications to `headers`.